apps/s_server.c

```diff
@@ -851,6 +851,8 @@ const OPTIONS s_server_options[] = {
      "Disable caching and tickets if ephemeral (EC)DH is used"},
     {"www", OPT_WWW, '-', "Respond to a 'GET /' with a status page"},
     {"WWW", OPT_UPPER_WWW, '-', "Respond to a 'GET with the file ./path"},
+    {"ignore_unexpected_eof", OPT_IGNORE_UNEXPECTED_EOF, '-',
+     "Use legacy error handling if there is no close_notify"},
```

Comment:
This text doesn't seem quite right. The error handling isn't like it was in legacy

Reply:
And the same applies to the option in s_client.

Reply:
Fixed both places. Thanks!